python/python/ert/enkf/data/gen_data.py

```diff
@@ -30,6 +30,18 @@ def getData(self):
         GenData.cNamespace().export_data( self , data )
         return data
 
+    def __getitem__( self, idx ):
+        ls = len(self)
+        if isinstance( idx, int ):
+            if idx < 0:
+                idx += ls
+            if 0 <= idx < ls:
+                return self.getData()[idx]
+            raise IndexError('List index out of range.')
+        if isinstance( idx, slice ):
+            vec = self.getData()
+            return [vec[i] for i in range(*idx.indices(ls))]
```

Comment:
With this implementation the slice return value will be a standard Python list; i.e. not the same type as self. I think that is the only possible way to implement slices for `GenData` - but I guess a comment could be added.